commands/logs.go

```diff
@@ -28,6 +31,10 @@ func init() {
 }
 
 func logsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	if cmd.Flags().Changed("format") || viper.GetString("format") == printer.FormatJSON {
+		return errors.New("format cannot be used")
```

Comment:
Suggesting a change on the text for clarity

```suggestion
		return errors.New("the \"--format\" flag cannot be used with this command")
```

Reply:
fixed